docs/content/tutorial.mdx

```diff
@@ -1,35 +1,50 @@
 ---
-title: Tutorials | Dagster
-description: Getting familiar with Dagster's feature set and tooling through hands-on tutorials.
+title: Intro to Software-defined assets | Dagster Docs
```

Comment:
💭

When I was testing this, I was stuck for a quick second since the Pre-Reqs/Setup section on this page is below the fold on my laptop (13")

My journey was:
1. Click on tutorial
2. See some text that I _should_ read but isn't vital to me getting set up. Scene setting and the outline. All very important in the pedagogy of teaching, but it's like asking a student to read the syllabus. The great students will read it and learn better, but it's an unrealistic expectation for an 18yo to read a syllabus.
3. Skip immediately to the "Defining an Asset" section
4. Get confused that we're jumping into code already and I haven't installed Dagster yet or set up my environment yet because I missed the "Pre-Requisites" section.

As a nudge, maybe at the top of the "Defining an Asset" page, we can link to the "Installation" page (though we shouldn't miss that the Tutorial comes with an additional dependency of `requests`).

What's your take on this?

Reply:
🌱 
This is an interesting point, and seems related to some other feedback I've seen about structuring tutorials. Some folks prefer a long, single page so they can skim and see everything in the tutorial at once, while others prefer the way this tutorial is set up - sections broken up into individual pages.

There are pros and cons to each approach:

- A single page puts everything in one place, allows users to use ctrl+f, and makes jumping around easier. On the other hand, the length can be intimidating.
- Separate pages could make it easier to define starting and stopping points, and also feel like you accomplished something. On the other hand, jumping around isn't as easy and you might miss something (like what happened here).

In the long term, I'd like to add a component to the docs that acts as a "you are here" indicator for tutorials/guides with multiple, lengthy steps. Prereqs/setup would always be included, so it would hopefully be easier to notice if something was skipped.

In the short term, we could:

- Move the Prereqs section to be first in the Defining an asset section
- Move the Prereqs section back to being its own page
- Keep the Prereqs section on this page, but move the syllabus/list down so setup shows above the fold
- Keep the Prereqs section on this page and roll the syllabus/list into it - ex:

   ```markdown
   In this tutorial, you'll:
   
   [list of things]
   
   To complete everything, you'll need:
   
   [setup list]
   ```

Reply:
I'll let you make the call on whether you want a short term change made, but if you're asking for my vote, I say 1 or 2!